examples/text-generation/requirements_awq.txt

```diff
@@ -1,3 +1,3 @@
 triton==3.1.0
-autoawq
-transformers>=4.48.2,<=4.49.0
+autoawq==0.2.7
+intel_extension_for_pytorch==2.6.0
```

Comment:
1. transformers are already in optimum-habana "main" dependencies, so they can be removed completely -- like in this PR
2. why do we want to introduce new dependency? (btw: isn't Intel Extension for PyTorch targeted mostly towards CPU and discrete GPUs?)